packages/config/src/memory/memory.config.ts

```diff
@@ -31,6 +32,20 @@ function isConfigTileSet(i: BaseConfig): i is ConfigTileSet {
   return ConfigId.getPrefix(i.id) === ConfigPrefix.TileSet;
 }
 
+/** Get the last id from the s3 path and compare to get the latest id based on the timestamp */
+function findLatestId(idA: string, idB: string): string {
+  const ulidA = ConfigId.unprefix(ConfigPrefix.Imagery, idA);
+  const ulidB = ConfigId.unprefix(ConfigPrefix.Imagery, idB);
+  try {
+    const timeA = decodeTime(ulidA);
+    const timeB = decodeTime(ulidB);
+    if (timeA >= timeB) return idA;
+  } finally {
+    //If not ulid return the return id alphabetically.
+    return idA.localeCompare(idB) ? idA : idB;
```

Comment:
I wonder if it would be cleaner to write `idA.localeCompare(idB) > 0 ? idA : idB`